src/Countries/US/Michigan/MichiganIncome/V20190101/MichiganIncome.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Michigan\MichiganIncome\V20190101;
+
+use Appleton\Taxes\Countries\US\Michigan\MichiganIncome\HasMichiganIncome;
+use Appleton\Taxes\Countries\US\Michigan\MichiganIncome\MichiganIncome as BaseMichiganIncome;
+
+class MichiganIncome extends BaseMichiganIncome
+{
+    // use HasMichiganIncome;
+    public const SUPPLEMENTAL_TAX_RATE = 0;
```

Comment:
I grab the supplemental rate from https://taxnews.ey.com/news/2019-0064-state-supplemental-income-tax-withholding-rates-for-2019. This might need to go in the ticket though.